### ISSUE TITLE
Perform a WCAG2.1 audit

### ISSUE THREAD

**Issue by cousquer** [cousquer/CardWebComponents#4]
Axe-core supports now WCAG 2.1 here is the result of its audit

![axe](https://user-images.githubusercontent.com/1481430/47780913-b5840e80-dcfc-11e8-8774-bde8f02aee77.png)

- Elements must have sufficient color contrast: 7 _Bootstrap doesn't meet AA level on links colors... 🤔 _
- Page must have one main landmark: 1 _Not Applicable: the main landmark should be in uPortal. won't fix_
- Page must contain a level-one heading: 1 _Not Applicable: the one level heading is in uPortal. won't fix_
- All page content must be contained by landmarks: 1 _Not Applicable: same... won't fix_

**Comment by cousquer** [comment]
resolved in #5
